posthog/migrations/0796_fix_revenue_analytics_modifiers_in_queries.py

```diff
@@ -0,0 +1,46 @@
+# Generated by Django 4.2.22 on 2025-07-16 16:38
+
+from django.db import migrations
+
+
+def remove_revenue_analytics_modifiers(apps, schema_editor):
+    Insight = apps.get_model("posthog", "Insight")
+
+    # Find all dashboard items that have the revenue analytics modifiers
+    items_to_update = Insight.objects.raw("""
+        SELECT * FROM posthog_dashboarditem
```

Comment:
For the unaware, this is correct `Insight` is the Python model, and `posthog_dashboarditem` is the correct table, check the `Metadata` info inside `Insight` for more info